.changeset/olive-moles-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/deno': patch
+---
+
+Fix Options types to include start. Types are extracted to a new file to avoid duplication.
```

Comment:
This becomes part of the changelog and gets shown to astro users. It would be helpful if it mentioned that it affects the config file.

Reply:
What about: Add `start` argument to the adapter's config type.